feat(Icon): add `Unlock` icon & use it in `DatasetHeader`

ramfox opened the pull request:
Also added instructions on how to import icons from the Figma board
to the `src/chrome/Icon.tsx` file

chriswhong reviewed (approved):
I think a yarn.lock change crept in here, FYI.  Everything else LGTM

ramfox committed:
feat(Icon): add Unlock icon & use it in DatasetHeader
Also added instructions on how to import icons from the Figma board
to the `src/chrome/Icon.tsx` file
